packages/braintree-integration/src/braintree-fastlane/is-braintree-fastlane-card-component.ts

```diff
@@ -0,0 +1,10 @@
+import {
+    BraintreeConnectCardComponent,
+    BraintreeFastlaneCardComponent,
+} from '@bigcommerce/checkout-sdk/braintree-utils';
+
+export default function isBraintreeFastlaneCardComponent(
+    cardComponent: BraintreeFastlaneCardComponent | BraintreeConnectCardComponent,
+): cardComponent is BraintreeFastlaneCardComponent {
+    return cardComponent.hasOwnProperty('getPaymentToken');
```

Comment:
Please add an extra condition for `render` method, since `BraintreeFastlaneCreditCardComponent` has 2 methods